sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestGetGeneratedKeysPostgres.java

```diff
@@ -83,4 +91,103 @@ public void testBatch() throws Exception {
             assertThat(dao.findNameById(ids[1])).isEqualTo("Macklin");
         });
     }
+
+    @Test
+    public void testUseRowMapperUpdate() throws Exception {
+        dbRule.getJdbi().useHandle(handle -> {
+            handle.execute("drop table something");
```

Comment:
We could move the code for recreating the `something` table to the `setUp` block and avoid duplication.